modules/rosapps/lib/vfdlib/vfdguisave.c

```diff
@@ -195,7 +195,11 @@ void OnInit(
 {
 	//	Store parameters
 
+#ifndef __REACTOS__
+	SetWindowLong(hDlg, GWL_USERDATA, (ULONG)pParam);
```

Comment:
may make sense to propose that fix upstream.

Reply:
I don't know any upstream. Anyway, ask the diff author, not me.

Reply:
I do understand the dilemma, ok then.